tests/e2e/cucumber/features/app-provider/officeSuites.feature

```diff
@@ -36,3 +36,33 @@ Feature: Integrate with online office suites like Collabora and OnlyOffice
     And "Anonymous" opens the public link "Link"
     And "Anonymous" unlocks the public link with password "%public%"
     Then "Anonymous" should see the content "Microsoft Word Content" in editor "OnlyOffice"
+
+
+  Scenario: update an OpenDocument file by an anonymous user with Collabora
+    Given "Alice" creates the following files into personal space using API
+      | pathToFile       | content                 |
+      | OpenDocument.odt | OpenDocument Content    |
+    And "Alice" creates a public link of following resource using the sidebar panel
+      | resource         | role     | password |
+      | OpenDocument.odt | Can view | %public% |
+    And "Alice" logs out
+    And "Anonymous" opens the public link "Link"
+    When "Anonymous" unlocks the public link with password "%public%"
+    Then "Anonymous" should not be able to edit the following resource
+      | resource         | type         | content                     |
+      | OpenDocument.odt | OpenDocument | Edited OpenDocument Content |
+
+
+  Scenario: update a Microsoft Word file by an anonymous user with OnlyOffice
+    Given "Alice" creates the following files into personal space using API
+      | pathToFile         | content                |
+      | MicrosoftWord.docx | Microsoft Word Content |
+    And "Alice" creates a public link of following resource using the sidebar panel
+      | resource           | role       | password |
+      | MicrosoftWord.docx | Can view   | %public% |
+    And "Alice" logs out
+    And "Anonymous" opens the public link "Link"
+    When "Anonymous" unlocks the public link with password "%public%"
+    Then "Anonymous" should not be able to edit the following resource
+      | resource           | type           | content                       |
+      | MicrosoftWord.docx | Microsoft Word | Edited Microsoft Word Content |
```

Comment:
can we extend the same journey in the above two scenarios instead of creating separate story?

Reply:
@ScharfViktor What is your thought on this?

Reply:
good point, I agree

```
Scenario: create an OpenDocument file with Collabora
    When "Alice" creates the following resources
      | resource         | type         | content              |
      | OpenDocument.odt | OpenDocument | OpenDocument Content |
    And "Alice" creates a public link of following resource using the sidebar panel
      | resource         | role     | password |
      | OpenDocument.odt | Can edit | %public% |
    And "Alice" logs out
    And "Anonymous" opens the public link "Link"
    And "Anonymous" unlocks the public link with password "%public%"
    Then "Anonymous" should be able to edit the following resource
      | resource           | type           | content                       |
      | MicrosoftWord.docx | Microsoft Word | Edited Microsoft Word Content |
      
    When "Alice" edits the public link named "Link" of resource "MicrosoftWord.docx" changing role to "Can view"
    And "Anonymous" opens the public link "Link"
    When "Anonymous" unlocks the public link with password "%public%"
    Then "Anonymous" should not be able to edit the following resource
      | resource           | type           | content                       |
      | MicrosoftWord.docx | Microsoft Word | Edited Microsoft Word Content |

```